File: luiz020/programacao_basica/Questao08.java
import java.util.Scanner;

public class Questao08 {

    public static void main (String [] args) {

        Scanner k;
        double a;
        double b;
        double c;
        double d;
        double e;
        double f;
        double g;
        double h;

        k = new Scanner (System.in);
        System.out.println(" Volume do Cubo ");
        System.out.println("Informe o lado do cubo: ");
        a = k.nextDouble ();
        b = a * a * a;
        System.out.println("O volume do cubo eh: " + b);

        System.out.print(" Volume do Paralelepípedo ");
        System.out.print("Informe a altura do paralelepípedo: ");
        c = k.nextDouble ();
        System.out.println("Informe o comprimento do paralelepípedo: ");
        d = k.nextDouble ();
        System.out.println("Informe a base do paralelepípedo: ");
        e = k.nextDouble ();
        f = c * d * e;
        System.out.println("O volume do paralelepípedo eh: " + f );

        System.out.print(" Volume da Esfera ");
        System.out.println("Informe o raio da esfera: ");
        g = k.nextDouble ();
        h = 4 * 3.14 * g * g * g/3;
        System.out.print("O volume da esfera eh: " + h);

        }}
 

File: luiz020/programacao_basica/questao08.py
print("Volume de um Cubo")
lado = float(input("Informe o lado do Cubo: "))
volume_cubo = lado * lado * lado
print("O volume do Cubo eh: " + str(volume_cubo))

print("Volume de um Paralelepípedo")
altura = float(input("Informe a altura do Paralelepípedo: "))
comprimento = float(input("Informe o comprimento do Paralelepípedo: "))
base = float(input("Informe a base do Paralelepípedo: "))
volume_para = altura * comprimento * base
print("O volume do Paralelepípedo eh: " + str(volume_para))

print("Volume de uma Esfera")
raio = float(input("Informe o raio da Esfera: "))
vol_esf = 4 * 3.14 * raio * raio * raio/3
print("O volume da Esfera eh: " + str(vol_esf))



